src/AbstractEnum.php

```diff
@@ -121,6 +123,21 @@
     }
 
     /**
+     * Asserts a constant with this name is defined.
+     *
+     * @param int|string $value  the value to test
+     * @param bool       $strict check the types of the value in the values
+     *
+     * @throws InvalidEnumName
+     */
+    final public static function assertValidName($name)
+    {
+        if (!self::isValidName($name)) {
```

Comment:
In this case maybe, but this change nothing using the keyword which covers every case IMHO. :wink: